contributor/03-contributor-ethos-article.md

```diff
@@ -21,56 +21,51 @@ knocking or ringing the door bell even if the door is open. Likewise in InnerSou
 as a visitor you won't be able (or invited) to commit directly to any
 code repository.
 
-Instead, when visiting your neighbors you're likely to at least knock on their
-door. Likewise in InnerSource after making your changes to the codebase you'll
+Instead, after making your changes to the codebase you'll
 submit them as a pull request. Much like you wouldn't go about making large
 changes and what you consider improvments to your neighbors home, in InnerSource
 you will anticipate and follow the project's collaboration guidelines. In
-turn your hosts will show you the way - in InnerSource that translates to
+turn, your hosts will show you the way - in InnerSource that translates to
 existing trusted committers spending their time to mentor guests.
 
-What about those lovely summer parties that you went to? Likely before going
-there was some planning ahead of time to choose the right date and time, to
-prepare enough food or have that contributed by the guests. The same happens for
-bigger changes in InnerSource projects: before making large modifications a
-project is likely to first ask you to submit an issue describing your need and
-your proposed solution. Spending some time on an initial design instead of
-jumping right into the implementation here saves contributors from having to
-re-do a lot of their work. Sharing progress early - even when it's not finished
-yet - helps mentoring the contributor towards a better solution. Much like
+What about those lovely summer parties that you went to? 
+There is usually some planning ahead of time to choose the right date and time, to
+prepare enough food, or to have it contributed by guests. The same happens for
+bigger changes in InnerSource projects: a project will likely ask you to submit 
+an issue describing your need and your proposed solution before making a large modification. 
+Spending time on initial design instead of
+jumping right into the implementation saves contributors from having to
+redo a lot of their work. Sharing progress early - even when it's not finished
+yet - helps the host team to mentor the Contributor towards a better solution. Much like
```

Comment:
I generally wondered if we could word that directly addressing the reader as in the contributor role. 
We did similar things in the earlier parts of the article I think.